### DISCUSSION TITLE
Slow load-times

### DISCUSSION THREAD

**Discussion by BlueSky-fur** [makeplane/plane#5148] [Q&A] [answered]
I'm encountering slow load-time when entering a issue-board.

Here's a video showing it better
https://github.com/user-attachments/assets/a3a20538-372a-41e4-8e2b-11058e47d8c5

My setup is as follows

- Ubuntu 24.04 LTS
- 4 cores dedicated vCPU
- 8 GB RAM
- 80GB NVMe

We also tested this setup on a high-end baremetal machine, with the same load-times.

These loadtimes are across multiple different end-machines, be it a MacBook, a Windows Laptop, a Windows PC, a Ubuntu desktop machine.

Any idea how to go about that issue?

**Comment by RKLBusinessDevelopment** [comment, marked as answer]
There already is a report for this issue (https://github.com/makeplane/plane/issues/5115). Many people face this issue, it seems to be caused by the pagination logic, which is currently being rewritten.

**Reply by BlueSky-fur** [reply]
Awesome! 

For whatever reason when i searched for loading issues, that issue did not pop up 🤔 Thanks for pointing me there!
